### PR TITLE
Yet more flake8 progress

### PR DESCRIPTION
**Pull request by silasary**
## What type of pull request is this?

- [x] Non-breaking code change
- [ ] Breaking code change
- [ ] Documentation change/addition

## Changes
- style: C816 missing trailing comma in Python 3.6+
- style: R503 missing explicit return at the  end of function ableto return non-None value.
- docs: Yet more return type annotations

## Checklist

- [x] I've formatted my code with [Black](https://black.readthedocs.io/en/stable/)
- [ ] I've ensured my code works on `Python 3.10.x`
- [ ] I've tested my code

### CODE REVIEW

File: dis_snek/models/discord/role.py [hunk 1]
```diff
@@ -16,8 +16,10 @@
     from dis_snek.models.discord.user import Member
     from dis_snek.models.discord.snowflake import Snowflake_Type
 
+T = TypeVar("T")
 
-def sentinel_converter(value, sentinel=attr.NOTHING):
+
+def sentinel_converter(value: bool | T, sentinel: T = attr.NOTHING) -> bool:
```
Comment (artem30801) [inline dis_snek/models/discord/role.py]: I think `value` should be also Optional as it can be None as per code

Comment (benwoo1110) [reply dis_snek/models/discord/role.py]: Looking at how attr converter works, a value should always be passed into a converter method. (anyways also not really the point of this pr?)

Comment (artem30801) [reply dis_snek/models/discord/role.py]: Optional == can be something OR `None`
And well, if this PR type hints this function, it better be more accurate?

Comment (benwoo1110) [reply dis_snek/models/discord/role.py]: yea i suppose so

Comment (silasary) [reply dis_snek/models/discord/role.py]: Good catch

**Pull request merged**